onebusaway-gtfs-transformer/src/main/java/org/onebusaway/gtfs_transformer/impl/VerifyFutureRouteService.java

```diff
@@ -98,12 +98,18 @@ int hasRouteServiceForDate(GtfsMutableRelationalDao dao, GtfsMutableRelationalDa
                 for (Trip refTrip : reference.getTripsForRoute(refRoute)) {
                     Set<ServiceDate> activeDates = refCalendarService.getServiceDatesForServiceId(refTrip.getServiceId());
                     if (activeDates.contains(sDate)) {
-                        _log.error("On {} Reference has service for this route but ATIS has none: {}, Trip {}, Serviceid {}",
-                                testDate, route.getId(), refTrip.getId(), refTrip.getServiceId());
-                        es.publishMessage(getTopic(), "Reference has service for this route but ATIS has none. Route: "
-                                + route.getId()
-                                + " has no service for "
-                                + testDate);
+                        if(route.getId().getId().equals("SUB_5X")){
```

Comment:
Let's abstract "SUB_5X" to a parameter or config that represents "problem routes" that will be dealt with differently